tests/serializers/test_python.py

```diff
@@ -61,6 +61,63 @@ class Parent(Object):
     )
 
 
+def test_and_parse_schema_with_description():
+    schema = {
+        "type": "object",
+        "title": "Parent",
+        "required": ["category"],
+        "properties": {
+            "category": {
+                "type": "object",
+                "title": "Category",
+                "properties": {
+                    "value": {
+                        "type": "string",
+                        "description": "value of category",
+                    }
+                },
+                "default": {"value": "none"},
+                "description": "category of parent",
+            },
+            "default": {"type": "string"},
+        },
+        "definitions": {
+            "other": {
+                "type": "object",
+                "title": "Other",
+                "properties": {"value": {"type": "integer"}},
+            }
+        },
+    }
+    assert serialize_python(*parse(schema)) == (
+        '''from statham.schema.constants import Maybe
+from statham.schema.elements import Integer, Object, String
+from statham.schema.property import Property
+
+
+class Other(Object):
+
+    value: Maybe[int] = Property(Integer())
+
+
+class Category(Object, default={'value': 'none'}, description='category of parent'):
+
+    """category of parent"""
```

Comment:
```suggestion
class Category(Object, default={'value': 'none'}):

    """category of parent"""
```

I would prefer if the generated classes only exposed the `description` field as a docstring. I'm hoping be possible in the metaclass (or maybe in `__init_subclass__`?) to parse the `__doc__` attribute and dynamically set the `description` attribute.

I think generally the logic when declaring a new Object sub-class should be:

1. If `description` kwarg is provided to the class, use this as `cls.description` (even if docstring is provided)
2. Otherwise, if a docstring is provided, set `cls.description = cls.__doc__`
3. Otherwise `cls.description = NotPassed()`

When generating code for `Object` classes, I think the serializer should always write to the docstring, and not pass the `description=` keyword argument.

Reply:
This does sound like a good solution, but to be honest, I don't think I'm familiar enough with how classes, inheritance and respective constructors to confidently implement this.